src/main/java/com/extrahardmode/config/RootNode.java

```diff
@@ -53,6 +53,22 @@
      */
     PRINT_COMMENTS("Print Comments", VarType.BOOLEAN, true, "If comments like this should be printed"),
     /**
+    * Globally disables certain modules. These settings might improve performance if these modules is not used. 
```

Comment:
Oh, I thought you were checking existing config options, not your new ones.

If you check to see if the respective options for the features are disabled in all worlds, perhaps you can "auto-disable" the module altogether without needing to create a new config option for it.